src/tiles/list/tile-list.ts

```diff
@@ -0,0 +1,374 @@
+  }
+
+  private get reviewsTemplate() {
+    return this.metadataTemplate(this.model?.commentCount, 'Reviews');
+  }
+
+  private get topicsTemplate() {
+    if (!this.model?.subjects || this.model.subjects.length === 0) {
+      return nothing;
+    }
+    return html`
+      <div id="topics" class="metadata">
+        ${this.labelTemplate('Topics')}
+        ${join(
+          map(this.model.subjects, id => this.searchLink('subject', id)),
+          html`, `
+        )}
+      </div>
+    `;
+  }
+
+  private get collectionsTemplate() {
+    if (!this.model?.collections) {
+      return nothing;
+    }
+    return html`
+      <div id="collections" class="metadata">
+        ${this.labelTemplate('Collections')}
+        ${join(
+          map(this.model.collections, id => this.detailsLink(id)),
+          html`, `
+        )}
+      </div>
+    `;
+  }
+
+  private get descriptionTemplate() {
+    return this.metadataTemplate(
+      DOMPurify.sanitize(this.model?.description ?? ''),
+      '',
+      'description'
+    );
+  }
+
+  // Utility functions
+  private metadataTemplate(text: any, label = '', id?: string) {
+    if (!text) return nothing;
+    return html`
+      <div id=${ifDefined(id)} class="metadata">
+        ${this.labelTemplate(label)} ${text}
+      </div>
+    `;
+  }
+
+  private labelTemplate(label: string) {
+    return html`${label
+      ? html`<span class="label">${label}: </span>`
+      : nothing}`;
+  }
+
+  private searchLink(field: string, searchTerm: string) {
+    if (!field || !searchTerm) {
+      return nothing;
+    }
+    const query = encodeURIComponent(`${field}:"${searchTerm}"`);
+    // No whitespace after closing tag
+    return html` <a href="${this.baseNavigationUrl}/search.php?query=${query}">
+      ${DOMPurify.sanitize(searchTerm)}</a
+    >`;
+  }
+
+  private detailsLink(identifier: string, text?: string) {
+    if (!identifier) {
+      return nothing;
+    }
+    const linkText = text ?? identifier;
+    // No whitespace after closing tag
+    return html` <a href="${this.baseNavigationUrl}/details/${identifier}"
+      >${DOMPurify.sanitize(linkText)}</a
+    >`;
+  }
+
+  /*
+   * TODO: fix field names to match model in src/collection-browser.ts
+   * private get dateSortSelector()
+   * @see src/models.ts
+   */
+  private get date(): Date | undefined {
+    switch (this.sortParam?.field) {
+      case 'date':
+        return this.model?.datePublished;
+      case 'reviewdate':
+        return this.model?.dateReviewed;
+      case 'addeddate':
+        return this.model?.dateAdded;
+      default:
+        return this.model?.dateArchived; // publicdate
+    }
+  }
+
+  private get classSize(): string {
+    return (this.currentWidth ?? 531) < 530 ? 'mobile' : 'desktop';
+  }
+
+  private get formatSize(): DateFormat | NumberFormat {
+    return (this.currentWidth ?? 531) < 530 ? 'short' : 'long';
+  }
+
+  static get styles() {
+    return css`
+      html {
+        font-size: unset;
```

Comment:
Does this affect the font size at the page level or just inside this component?

Reply:
just inside this component